source/developer/integration-faq.md

```diff
@@ -41,9 +41,24 @@ For self-hosted deployments in small setups you might host integrations on the s
 
 When self-hosting restrictions are less strict, AWS, Heroku and other public cloud options could also be used.
 
-## How do I create a bot account without using webhooks?
+## How do I create a bot account with personal access tokens?
 
-Deployments that cannot create bot accounts via webhooks due to security reasons can use the following approach:
+You can use the following approach to create a bot account with personal access tokens:
```

Comment:
Non-blocking and 0/5 but this line seems redundant since the question contains the exact same information really